packages/compass-indexes/src/components/regular-indexes-table/index-actions.tsx

```diff
@@ -58,6 +64,19 @@ const IndexActions: React.FunctionComponent<IndexActionsProps> = ({
       );
     }
 
+    // you can only drop regular indexes or failed inprogress indexes
```

Comment:
This was also a bug - you could drop an in progress index that was really still in progress and that just wouldn't work because the index creation isn't failed and it isn't a real index yet either.